app/Http/Kernel.php

```diff
@@ -18,6 +18,7 @@ class Kernel extends HttpKernel
         \Illuminate\Foundation\Http\Middleware\ValidatePostSize::class,
         \App\Http\Middleware\TrimStrings::class,
         \Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull::class,
+        \Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse::class,
```

Comment:
You should probably open a PR to update the docs in Laravel for this ? 😛

Reply:
Well no apparently this is wanted as it was. And actually as I don't really need it anymore I'll remove this change.